### ISSUE TITLE
Controller's get method is not respected

### ISSUE THREAD

**Issue by malarzm** [lookyman/phpstan-symfony#9]
First, thanks for the plugin! :)

Controller:

```
abstract class BaseController extends \Symfony\Bundle\FrameworkBundle\Controller\Controller
{
    protected function handleCommand($command)
    {
        $this->get('command_bus')->handle($command);
    }
}
```

And after analyse I'm getting for the controller:

> Call to an undefined method object::handle().

Let me know should I try to hack a failing test case or is this enough.

**Comment by malarzm** [comment]
This is generated in compiled XML: `<service id="command_bus" class="....">`

**Comment by malarzm** [comment]
Probably it's worth mentioning, I'm running Symfony 3.2 ATM

**Comment by malarzm** [comment]
After a bit of hacking I've realized that the inspection fails due to usage:

```
$this->get('command_bus')->handle($command);
```

fails, why this:

```
$this->container->get('command_bus')->handle($command);
```

will pass.

**Comment by lookyman** [comment]
Yep, so far it only checks Container, not Controller. I plan to add support for it.

**Comment by malarzm** [comment]
If it should resemble what is done for container, I can send you a PR

**Comment by lookyman** [comment]
Fixed by 8e8fa73.
